core/new-gui/src/app/workspace/component/property-editor/versions-display/versions-display.component.ts

```diff
@@ -53,7 +60,7 @@ export class VersionsListDisplayComponent implements OnInit {
     this.retrieveVersionsOfWorkflow(wid)
       .pipe(untilDestroyed(this))
       .subscribe(versionsList => {
-        this.versionsList = versionsList;
+        this.versionsList = versionsList.reverse();
```

Comment:
I understand it is a simple hack on the frontend. but maybe it's more natural to change the backend to output versions in a reverse order directly from DB. in the future we might want to do pagination when versions are huge, and the order of pagination query should match the UI.

Reply:
done